newsfragments/5044.added.md

```diff
@@ -0,0 +1 @@
+Implement `OnceExt` & `MutexExt` for `parking_lot` & `lock_api`
```

Comment:
You should also mention the new features IMO

Reply:
Reading over this again, I realize this may be confusing. I was referring to the new cargo features.